src/main/java/org/sagebionetworks/web/client/widget/entity/MarkdownWidgetViewImpl.java

```diff
@@ -88,10 +100,10 @@ private void addPlaceChangerEventHandlerToAnchors() {
     NodeList<Element> anchors = contentPanel
       .getElement()
       .getElementsByTagName("a");
-    String hostPageURL = GWT.getHostPageBaseURL();
+    String hostPrefix = gwt.getHostPrefix();
```

Comment:
This is code that detects relative hrefs.  If the hostPrefix matches, then it's pointing to a local resource.